packages/driver/src/utils.ts

```diff
@@ -20,6 +20,13 @@ import type { ResolvedConnectConfigReadonly } from "./conUtils";
 import type { HttpSCRAMAuth } from "./httpScram";
 import type { ProtocolVersion } from "./ifaces";
 
+export {
+  Float16Array,
+  getFloat16,
+  isFloat16Array,
+  setFloat16,
+} from "@petamoriken/float16";
```

Comment:
```suggestion
} from "npm:@petamoriken/float16";
```

Reply:
Ignore this. It's a dependency in package.json, so it should work without the npm prefix.

Reply:
yes, and I'm quite sure this don't work with `npm:` prefix in Node